Add link to combat instructions for solo/coop scenarios

captain-yoshi opened the pull request:
Fixes #728.

![en-31](https://github.com/user-attachments/assets/250032de-93e0-4e50-9c15-5f48ca9571dd)

qwrtln commented on sections/combat.tex:
```diff
@@ -46,6 +46,7 @@ \subsection*{\pagetarget{Combatsetup}{Combat Setup}}
 Units must be placed in \textbf{descending} Initiative\index{Initiative} order.
 If there's a tie, place higher tier Units first.
 If there's still a tie, the players decide the order.
+Check the \hyperlink{combatai}{\textbf{Combat against AI}} section for combat instructions.
```
```suggestion
Check the \pagelink{combatai}{\textbf{Combat against AI}} section for combat instructions.
```

qwrtln replied:
No need for a new link, reuse the existing one.

qwrtln commented on sections/ai_rules.tex:
```diff
@@ -66,7 +66,7 @@ \subsection*{AI Decks}
 If an AI Hero is instructed to draw a card, they will draw and resolve \textbf{another card} from the AI Deck.
 
-\subsection*{Combat against AI}
+\subsection*{\pagetarget{combatai}{Combat against AI}}
```
```suggestion
\subsection*{Combat against AI}
```
No need

captain-yoshi commented:
Rebased with master. Is there something to add for localisation ?

qwrtln commented:
> Rebased with master. Is there something to add for localisation ?

Yes, you have to run `po4a po4a.cfg`. It will update the localization files.

captain-yoshi commented:
Got this errror when running:
```bash
$ po4a po4a.cfg
sections/ai_rules.tex:53: (po4a::tex)
               unmatched end of environment 'multicols*'
po4a: Couldn't determine the input document's charset. Please specify it on the command line. (non-ASCII char at sections/all_map_locations.tex:38)
```

The non-ASCII char is the em dash from  `all_map_locations.tex` at line 38
```latex
    \svg{gold}/\svg{building_materials}/\svg{valuablegreater} —
```
What would be the correct charset to use ?

qwrtln commented:
We're using Unicode here, since LaTeX supports it at least since 2018. The workflow uses version po4a==0.73. If your distro ships with po4a<=0.69, you might encounter problems. You have two options:
1) Download the latest po4a from GitHub: https://github.com/mquinson/po4a
2) Create a PR in your fork targeting `combat-setup` branch, and type a comment `po4a commit` in there. A bot in the workflow will commit the necessary changes. After merging them, they'll be reflected here.

qwrtln commented:
Charset fixed by #731

qwrtln commented:
It shouldn't have changed that many files. As #731 was merged, try removing the latest commit, rebasing, and running again.

qwrtln commented:
You probably still have local po4a version mismatch (which one do you have, BTW?). Try the workflow approach.

captain-yoshi commented:
Removed, rebased and run `po4a cfg`. It still seem to change a lot of files Using version po4a version 0.74-alpha. Maybe I should use a release version instead.

qwrtln commented:
0.73

captain-yoshi commented:
Had an error when building po4a but showed new version when installed...:
```bash
Malformed encoding while writing to file /home/captain-yoshi/ws/dev/po4a/blib/man/ca/man3/Locale::Po4a::Xml.3pm.pod with charset UTF-8: "\x{fffd}" does not map to UTF-8 at lib/Locale/Po4a/TransTractor.pm line 
542.
If UTF-8 is not the expected charset, you need to configure the right one with with --localized-charset or other similar flags.
Close with partial character at lib/Locale/Po4a/TransTractor.pm line 566.
Died at Po4aBuilder.pm line 191.
```
Anyway generated the localisation update through the Bot and rebased.

captain-yoshi commented:
The CS printing gives 57 pages instead of 56 which might be normal. The strange thing is that this PR section has not been translated ?
![image](https://github.com/user-attachments/assets/1422102f-b0d3-4fa6-8ad2-a208e64869a4)

qwrtln commented:
That's what po4a does. If there's an update to a part of the text (the bullet points in this case), it marks the translation as fuzzy, and uses the original as fallback. Every language will do that, that's why your change is rolled out to 8 po files. I think it should be parked until 1.3 release, as it's not a critical change, so as not to trouble translators.

qwrtln commented:
Please update and reopen if still relevant.
